working-together/README.md

```diff
@@ -156,6 +156,11 @@ You can [download a double-sided printout] of these guides to display in your me
   - Let everyone know ahead of time what they can expect.
   - Create a calendar invite with links to the video call and dial-in information (if applicable)
   - Create a meeting agenda or note about what the meeting will cover.
+  - Strongly avoid vague meeting invitations. For people with anxiety, vague
+    invitations can cause huge amounts of stress or
+    [panic attacks](https://en.wikipedia.org/wiki/Panic_attack).
+    - If you're incapable of sharing an overview or topic, at least share the
```

Comment:
"incapable" might imply someone doesn't know how, so maybe "unable" is better, given the further explanation?

Another thought: it is possible to make calendar events (at least on Google Calendar) private, so an uninvited colleague shouldn't be able to see it.